NavKit/CustomizableNavigation.swift

```diff
@@ -38,6 +38,10 @@ public protocol CustomizableNavigation {
     /// Specify this property to determine whether or not the screen could be
     /// dragged from left to right to go to the previous screen.
     var isUsingInteractivePopGesture: Bool { get }
+    
+    /// Specify this property to determine whether or not the navigation bar
+    /// title is large or not
```

Comment:
Thanks for specifying the comments clearly! 👍🏻 Just a small matter to improve consistency, could you please add a full stop at the end?

Reply:
Yup all good!